src/Sulu/Bundle/AdminBundle/Admin/View/ViewRegistry.php

```diff
@@ -115,14 +123,14 @@ private function mergeViewOptions(array $views, string $parent = null)
             return [];
         }
 
-        /** @var View $parentView */
+        /** @var View[] $parentViews */
         $parentViews = \array_values(\array_filter($views, function(View $view) use ($parent) {
             return $view->getName() === $parent;
         }));
 
         $parentView = null;
         if (!empty($parentViews)) {
-            $parentView = $parentViews[0];
+            $parentView = \current($parentViews);
```

Comment:
not sure if it makes a difference in this case, but `current` does not always return the first element. i think we should use `reset` if it is necessary to change this

Reply:
maybe just `list($parentView) = $parentViews;

Reply:
looks like `list($test)` will trigger a `Undefined variable` notice if the array is empty. Therefore I would use `reset` 🙂 
https://3v4l.org/Cg2bs